src/vs/workbench/contrib/userDataProfile/browser/userDataProfilesEditorModel.ts

```diff
@@ -786,7 +786,9 @@ export class NewProfileElement extends AbstractUserDataProfileElement {
 	}
 
 	private async getChildrenFromProfileTemplate(profileTemplate: IUserDataProfileTemplate, resourceType: ProfileResourceType): Promise<IProfileResourceTypeChildElement[]> {
-		const profile = toUserDataProfile(generateUuid(), this.name, URI.file('/root').with({ scheme: USER_DATA_PROFILE_TEMPLATE_PREVIEW_SCHEME }), URI.file('/cache').with({ scheme: USER_DATA_PROFILE_TEMPLATE_PREVIEW_SCHEME }));
+		const location = URI.file(`/root/profiles/${profileTemplate.name}`).with({ scheme: USER_DATA_PROFILE_TEMPLATE_PREVIEW_SCHEME });
```

Comment:
fyi - you don't need to do `URI.file(...).with(..)` and that you can use `URI.from({scheme, path})`

Reply:
Thanks. ✅